src/ParseObject.js

```diff
@@ -935,10 +939,22 @@ class ParseObject {
   }
 
   /**
-   * Clears any changes to this object made since the last call to save()
+   * Clears any (or specific) changes to this object made since the last call to save()
+   * @param {String|Array<String>|...String} [keys] - specify which fields to revert
    */
-  revert(): void {
-    this._clearPendingOps();
+  revert(...keys?: Array<string>): void {
+    let _keysToRevert;
+    if(keys.length) {
```

Comment:
This will throw an exception if keys is undefined. Please address

Reply:
As for now it doesn’t throw an exception but I’m genuinely curious as for why. Can you explain?

Reply:
Not at all,  the `?` never sees the light of the Babel transformation as it’s stripped out of the source code first.

The reason is just because ...keys is simply the same as doing `keys = Array.from(arguments)`

Reply:
Ah okay, got it, you're right - I can add an empty check regardless to avoid future confusion

Reply:
It’s not that, it’s just that calling with not arguments or an empty array will have different meanings and results, as in the latter case the length will be 1 and it may have additional side effects